### PR TITLE
Notabene patch 1 footer navigation

### PR DESCRIPTION
**Pull request by notabene**
Improving the Accessiibility label in the footer.

### CODE REVIEW

**Comment by notabene** [comment]
@Orange-OpenSource/accessibility needs review please :) (good first review)

File: src/_data/navigation.js [hunk 1]
```diff
@@ -199,14 +199,14 @@ module.exports = {
       { label: 'Contact', href: '/en/contact/' },
       { label: 'About', href: '/en/about/' },
       { label: 'RSS feed', href: '/feed-en.xml' },
-      { label: 'Accessibility', href: '/en/accessibility/' },
+      { label: 'Accessibility:partially compliant', href: '/en/accessibility/' },
```
Comment (MewenLeHo) [inline src/_data/navigation.js:202]:
```suggestion
      { label: 'Accessibility: partially compliant', href: '/en/accessibility/' },
```

**Review by MewenLeHo — CHANGES_REQUESTED** [review]
Almost good, just a small fix to add a missing space and we will be all set!

**Review by MewenLeHo — APPROVED** [review]
LGTM 🚀

**Pull request merged**